aengusmcmillin/gatsby-theme-brain
# setup exclusion list

Fixes #32

## What 
- Adds exclusion option to the plugin
- Files matching exclusions are not included in the build when getting notes
- Exclusions can be strings or regexps

## How to test
- Set up a gatsby project using this project
- Add 3 notes to your content directory (`note-a.mdx`, `note-b.mdx`, `note-c.mdx`)
- Add `exclude: [/^note-b/, 'note-c.mdx']` to your plugin options
- When you run the build, a page should be generated only for `note-a.mdx`
